cfme/tests/v2v/test_v2v_manual.py

```diff
@@ -0,0 +1,632 @@
+        assignee: ytale
+        casecomponent: V2V
+        caseimportance: medium
+        initialEstimate: 1/8h
+        startsin: 5.10
+        subcomponent: OSP
+        testSteps:
+            1. Create infrastructure mapping for vmware to OSP
+            2. Create migration plan with VM with multiple nics
+            3. Start migration
+        expectedResults:
+            1.
+            2.
+            3. Successful migration
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_osp_test_retry_plan():
+    """
+    OSP: Test retry plan
+
+    Polarion:
+        assignee: ytale
+        casecomponent: V2V
+        initialEstimate: 1/8h
+        startsin: 5.10
+        subcomponent: OSP
+        testSteps:
+            1. Create infrastructure mapping for vmware to OSP
+            2. Create migration plan so that it fails
+            3. Retry migration
+        expectedResults:
+            1.
+            2.
+            3. Migration starts
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_osp_test_user_can_download_post_migration_ansible_playbook_log():
+    """
+    title: OSP: Test user can download post migration ansible playbook log
+
+    Polarion:
+        assignee: ytale
+        casecomponent: V2V
+        initialEstimate: 1/8h
+        startsin: 5.10
+        subcomponent: OSP
+        testSteps:
+            1. Create infrastructure mapping for vmware to OSP
+            2. Create migration plan and run ansible playbooks
+            3. Start migration
+        expectedResults:
+            1.
+            2.
+            3. Post migration logs should be downloadable
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_osp_test_vm_owner_before_and_after_migration_remains_same():
+    """
+    title: OSP: Test VM owner before and after migration remains same
+
+    Polarion:
+        assignee: ytale
+        casecomponent: V2V
+        caseimportance: medium
+        initialEstimate: 1/8h
+        startsin: 5.10
+        subcomponent: OSP
+        testSteps:
+            1. Create infrastructure mapping for vmware to OSP
+            2. Create migration plan with a VM with some owner
+            3. Start migration
+        expectedResults:
+            1.
+            2.
+            3. VM owner should remain same
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_osp_test_migrating_a_vm_which_has_encrypted_disk():
+    """
+    title : OSP: Test migrating a VM which has encrypted disk
+
+    Polarion:
+        assignee: ytale
+        casecomponent: V2V
+        initialEstimate: 1/8h
+        startsin: 5.10
+        subcomponent: OSP
+        testSteps:
+            1. Create infrastructure mapping for vmware to OSP
+            2. Create migration plan with a VM with encrypted disk
+            3. Start migration
+        expectedResults:
+            1.
+            2.
+            3. Successful migration
+    """
+    pass
+
+
+@pytest.mark.tier(2)
+def test_osp_vmware60_test_vm_with_multiple_disks():
```

Comment:
RHV usecase automated already, same can be used for OSP too, remove this since we are fixing OSP failures next week  -> `test_dual_datastore_dual_vm_migration`